examples/include-ext.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Modified from ext-5.0.0/shared/include-ext.js
```

Comment:
They were minor:

* support query parameter 'extjs'
* default to 'neptune' theme, as this one exists in both `v4` and `v5`
* load the `debug`-variants of the resources

I'll add a commit explaining the differences.